### ISSUE TITLE
winpty ptyprocess error

### ISSUE THREAD

**Issue by Braumasta** [faroukbmiled/BOIIIWD#63]
![Screenshot 2024-07-30 164650](https://github.com/user-attachments/assets/9023e0b9-feec-4947-be56-194ef7d93cc3)
I am having this issue.

**Comment by Braumasta** [comment]
hi I keep getting this exception when trying to download a custom map

**Comment by Braumasta** [comment]
nvm i downloaded v0.3.5 and i can download the map

**Comment by faroukbmiled** [comment]
> ![Screenshot 2024-07-30 164650](https://github.com/user-attachments/assets/9023e0b9-feec-4947-be56-194ef7d93cc3)
> 
> I am having this issue.

If you don't mind me asking ,Can you tell me what cpu you have? Or which laptop you using

**Comment by Braumasta** [comment]
hi sorry for late response I have the i7 12650h cpu im on a gaming laptop

**Comment by Braumasta** [comment]
my bad i closed by accident

**Comment by Braumasta** [comment]
but i used an older version of the workshop downloader and was able to download the maps

**Comment by faroukbmiled** [comment]
> but i used an older version of the workshop downloader and was able to download the maps

Yeah that's good ,but it can happen for other ppl as well, one last thing can you share a screen shot of the 'About your pc' just like this one, and thank you

![image](https://github.com/user-attachments/assets/0551edb7-3bb9-40a6-8602-67a6bde8e9b8)

**Comment by Braumasta** [comment]
hi sorry for the late response I haven't been on in 3 days here is a picture.
![Screenshot 2024-08-05 033734](https://github.com/user-attachments/assets/254d7bf9-b1ae-4262-b877-bf2bf1b6c001)
